build.rs

```diff
@@ -0,0 +1,54 @@
+mod cli {
+    include!("src/cli.rs");
+}
+
+const LONG_ABOUT: &str =
+    "prr is a tool that brings mailing list style code reviews to Github PRs. This \
+means offline reviews and inline comments, more or less.
+
+To that end, prr introduces a new workflow for reviewing PRs:
+  1. Download the PR into a \"review file\" on your filesystem
+  2. Mark up the review file using your favorite text editor
+  3. Submit the review at your convenience
+
+For full documentation, please visit https://doc.dxuuu.xyz/prr/.";
+
+fn main() -> std::io::Result<()> {
+    if let Some(out_path) = std::env::var_os("GEN_DIR").or(std::env::var_os("OUT_DIR")) {
```

Comment:
I've only seen `OUT_DIR` before -- what is `GEN_DIR` used for?

Reply:
`GEN_DIR` is an ptionally specified directory to put the generated manpages and completions, since `OUT_DIR` (which is set by Cargo) is somewhere in the target/ directory and can be hard to reach.